monai/losses/perceptual.py

```diff
@@ -170,9 +176,9 @@ def forward(self, input: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
             loss = loss_sagittal + loss_axial + loss_coronal
         else:
             # 2D and real 3D cases
-            loss = self.perceptual_function(input, target)
+            loss = self.perceptual_function(input, target).squeeze()
 
-        return torch.mean(loss)
+        return torch.mean(loss, dim=0)
```

Comment:
May I know the reason for these change? The squeeze here is for? And the loss will alway keep the channel dim?

Reply:
Hi @SomeUserName1, could you please help explain more here?
Seems the result will always be [1*C*1]？

Reply:
Ah sorry, I just figured, that pending means I have to submit a review in files changed...

Reply:
Done